services/QuillLMS/client/app/assets/styles/home.scss

```diff
@@ -189,17 +189,17 @@ a.q-button {
 $list: (quillgreen: $quillgreen, quillred: $quillred, quillnavbar: $quillnavbar, quillforestgreen: $quillforestgreen, quillblue: $quillblue, quillteal: $quillteal, white: $white, black: $black, orange: $orange, red: $red, darkred: $darkred, darkorange: $darkorange, darkblue: $darkblue, green: $green, darkgreen2: $darkgreen2, lightgray: $lightgray, darkgray: $darkgray, purple: $purple, lessonspurple: $lessonspurple);
 
 @each $name, $color in $list {
-  .text-#{$name} {
+  .text-#{"" + $name} {
```

Comment:
Some sort of type coercion?

Reply:
That's right. There's an in-built SASS warning about interpolation of color names which was causing our builds to fail. This sidesteps that.